### PR TITLE
Fix/test awss3 helper

### PR DESCRIPTION
**Pull request by VasilHamulyak**
## JIRA

* [Main JIRA ticket](https://ssu-jira.softserveinc.com/secure/RapidBoard.jspa?rapidView=777)

## Code reviewers

- [ ] @ashatna
- [ ] @ChristinaPastushchak
- [ ] @dotiurin
- [ ] @iryn34
- [ ] @tarassito
- [ ] @VasilHamulyak
- [ ] @VitalyiKhlivnyi
- [ ] @YurijH

### Second Level Review

- [ ] @ushevtsiv
- [x] @lhalam
- [ ] @ssOleg

## Summary of issue

ToDo

## Summary of change

ToDo

## Testing approach

ToDo

## CHECK LIST
- [ ]  СI passed
- [ ]  Сode coverage >=95%
- [ ]  PR is reviewed manually again (to make sure you have 100% ready code)
- [ ]  All reviewers agreed to merge the PR
- [ ]  I've checked new feature as logged in and logged out user if needed
- [ ]  PR meets all conventions

### CODE REVIEW

**Review by lhalam — APPROVED** [review]

File: eventually/tests/unittests/utils/test_awss3_helper.py [hunk 1]
```diff
@@ -0,0 +1,193 @@
...
+IMAGE_NAME = "testimage.png"
+IMAGE_MIME_TYPE = "image/png"
+GOOD_IMAGE_SIZE = 3 * 1024 * 1024
+BAD_IMAGE_SIZE = 9 * 1024 * 1024
+IMAGE_FORMAT = "png"
+BUCKET_ADDRESS = "https://s3.eu-west-2.amazonaws.com/eventually-photos/"
+
+
+class MockObjects():
+    key = "image_key"
```
Comment (lhalam) [inline eventually/tests/unittests/utils/test_awss3_helper.py]: move to constant

File: eventually/tests/unittests/utils/test_awss3_helper.py [hunk 2]
```diff
@@ -0,0 +1,193 @@
...
+
+IMAGE_NAME = "testimage.png"
+IMAGE_MIME_TYPE = "image/png"
+GOOD_IMAGE_SIZE = 3 * 1024 * 1024
+BAD_IMAGE_SIZE = 9 * 1024 * 1024
+IMAGE_FORMAT = "png"
+BUCKET_ADDRESS = "https://s3.eu-west-2.amazonaws.com/eventually-photos/"
+
+
+class MockObjects():
```
Comment (lhalam) [inline eventually/tests/unittests/utils/test_awss3_helper.py]: rename `MockObjects `-> `MockObjectsLoadRaiseClientError`

File: eventually/tests/unittests/utils/test_awss3_helper.py [hunk 3]
```diff
@@ -0,0 +1,193 @@
...
+
+class MockBOTO_S3():
+    """ Class for simulate BOTO_S3 """
+
+    @staticmethod
+    def Object(*args, **kwargs):
+        return MockObjects()
+
+
+class MockObjectsTrue():
```
Comment (lhalam) [inline eventually/tests/unittests/utils/test_awss3_helper.py]: rename `MockObjects`-> `MockObjectsLoadReturnTrue`

**Review by lhalam — DISMISSED** [review]

**Review by VitalyiKhlivnyi — APPROVED** [review]

**Review by lhalam — APPROVED** [review]
Good job!

**Pull request merged**